<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>60</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Progress</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
        <widget class="QLabel" name="infoText">
            <property name="text">
            <string>Progress</string>
            </property>
        </widget>
        </item>
        <item>
        <widget class="QProgressBar" name="progressBar">
            <property name="value">
            <number>0</number>
            </property>
            <property name="textVisible">
            <bool>true</bool>
            </property>
            <property name="invertedAppearance">
            <bool>false</bool>
            </property>
        </widget>   
    </item>     
     <item>
      <layout class="QVBoxLayout" name="PlotVerticalLayout"/>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
